packages/server/src/api/storage/sql/build-model.js

```diff
@@ -24,9 +24,11 @@ const attributes = {
   avatarUrl: {type: Sequelize.STRING(256)},
   ancestorHash: {type: Sequelize.STRING(40)},
   externalBuildUrl: {type: Sequelize.STRING(256)},
-  runAt: {type: Sequelize.DATE()}, // should mostly be equal to createdAt but modifiable by the consumer
-  committedAt: {type: Sequelize.DATE()},
-  ancestorCommittedAt: {type: Sequelize.DATE()},
+  runAt: {type: Sequelize.DATE(6)}, // should mostly be equal to createdAt but modifiable by the consumer
```

Comment:
these are all because of https://dev.mysql.com/doc/refman/8.0/en/fractional-seconds.html, right?

> The fsp value, if given, must be in the range 0 to 6. A value of 0 signifies that there is no fractional part. If omitted, the default precision is 0. (This differs from the standard SQL default of 6, for compatibility with previous MySQL versions.)

Reply:
Yes else mysql will not save any milliseconds

Reply:
You can also see a super short comment on the "Data types" at the Sequelize documentation:
https://sequelize.org/v4/manual/tutorial/models-definition.html
(But yes the mysql site gives more information about the problem)